README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project that I learned about linter, HTML and css
+
+## Built With
+
+- HTML
+- Css
+
+
+## Getting Started
+
+**This is the first project using HTML and css and I learned about linter**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
Remove all unnecessary sections in your README file in order to show a professional README file.

- [ ] Remove empty sections from your README file.